tds/src/main/java/thredds/server/wfs/WFSController.java

```diff
@@ -88,7 +88,24 @@ private void describeFeatureType(PrintWriter out, HttpServletRequest hsreq) {
 	 * @return
 	 */
 	private void getFeature(PrintWriter out, HttpServletRequest hsreq) {
-		WFSGetFeatureWriter gfdw = new WFSGetFeatureWriter(out, WFSController.constructServerPath(hsreq), WFSController.getXMLNamespaceXMLNSValue(hsreq));
+
+		//test out functionality for different geometries

```

Comment:
This is a good way to demo the different SimpleGeometry types! As we build out the getFeature work, keep in mind that the geometry types in our GetFeature responses must match the GML types described in the DescribeFeatureType responses.  I don't think they match in this PR, but this is a good showcase of the new functionality, and we'll likely be changing this soon so I'd say still quite mergeworthy.